index.bs

```diff
@@ -310,30 +310,45 @@ band communication channel between devices.
 Sensors can potentially be used in cross-device linking and tracking of a user.
 
 
-<h3 id="browsing-context">Browsing Context</h3>
+<h3 id="mitigation-strategies">Mitigation Strategies</h3>
+
+
+<h4 id="secure-context">Secure Context</h4>
+
+[=Sensor readings=] are explicitly flagged by the
+Secure Contexts specification [[POWERFUL-FEATURES]]
+as a high-value target for network attackers.
```

Comment:
Well, secure context is precisely a response to [network attackers](https://w3c.github.io/webappsec-secure-contexts/#threat-models-risks) as defined in the secure contexts spec, so that seems appropriate, no?